src/components/Table/Table.tsx

```diff
@@ -44,17 +43,19 @@ export type Row = {
 export type TableProps = PropsWithChildren<{
     columns: Column[];
     rows: Row[];
-    onSelectionChange?: (ids?: (string | number)[]) => void;
+    onSelectionChange?: (ids?: Key[]) => void;
     selectionMode?: SelectionMode;
     selectedRowIds?: (string | number)[];
     ariaLabel?: string;
 }>;
 
 const DEFAULT_SORT_ORDER = 'descending';
 
+export type SortDirection = 'ascending' | 'descending';
+
 type SortType = {
     sortedColumnKey?: string | number;
```

Comment:
It would make sense to use the `Key` type here as well. Also on type `Row` `key: Key` and other places. don't you think?

Reply:
Total sense 👍